pkgs/tools/misc/apkeep/default.nix

```
@@ -12,7 +12,9 @@ rustPlatform.buildRustPackage rec {
   cargoSha256 = "sha256-YFs2AOMGp0WNrceK14AnigZdJl+UsQdUchpxaI7HSXw=";
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ openssl ];
+
+  buildInputs = lib.optionals stdenv.isLinux [ openssl ]
```

It still needs `openssl` according to the failure: https://github.com/NixOS/nixpkgs/pull/145452/checks?check_run_id=4180402354
```suggestion
  buildInputs = [ openssl ]
```

Tested it locally, and it builds correctly and runs with this change.